README.md

```diff
@@ -216,19 +220,22 @@ wnbd-client.exe list
 ```
 ```
 Pid         DiskNumber  Nbd    Owner            InstanceName
-3508        1           true   wnbd-client      test2
+3508        1           true   wnbd-client      foo
 4024        2           false  ceph-rbd-wnbd    rbd/rbd_win_10g
 ```
 
 ### Unmapping the device
 
+By default, a soft disconnect is attempted. If that fails or times out,
```

Comment:
Does it make sense to emphasize what a soft and hard disconnect is?

Reply:
I guess adding a few words on that wouldn't hurt.